stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/ClientCall.java

```diff
@@ -185,7 +186,8 @@ public void sendMessage(Message message) {
         try {
             InputStream resp = method.streamRequest(message);
             outboundMessage.sendMessage(resp);
-
+        } catch (StatusRuntimeException ex) {
```

Comment:
Why we need to catch this if we don't do anything? Since this is a RuntimeException shall we leave without handling it?

Reply:
Earlier for any exception we throw StatusRuntimeException and we need to avoid reconstructing exception if it is StatusRuntimeException.